lib/cube_browser/__init__.py

```diff
@@ -0,0 +1,48 @@
+import matplotlib.pyplot as plt
+
+
+class Contourf(object):
+    def __init__(self, cube, coords, **kwargs):
+        self.cube = cube
+        self.coords = coords # coords to plot
+        self.kwargs = kwargs
+        self.axes = self.new_axes()
+        self.qcs = None  # QuadContourSet
+        # A mapping of 1d-coord name to dimension
+        self.coord_dim = self.coord_dims()
+
+    # This constructs the original and updated plots
+    def __call__(self, **kwargs):
```

Comment:
i'm a little wary of this function taking kwargs i think i prefer taking one argument, which is iterable, and not using kwargs.items()

@bjlittle 's opinion is really helpful here